web/locale/en_US/messages.po

```diff
@@ -2696,17 +2696,17 @@ msgstr "No invites found"
 msgid "How do I choose?"
 msgstr "How do I choose?"
 
-msgid "Champion - A CoderDojo Champion is an individual who spearheads setting up and maintaining a Dojo. Champions do not necessarily have to have the ability to code but possess the skills required to bring together technical mentors and supporters to run a Dojo frequently!"
-msgstr "Champion - A CoderDojo Champion is an individual who spearheads setting up and maintaining a Dojo. Champions do not necessarily have to have the ability to code but possess the skills required to bring together technical mentors and supporters to run a Dojo frequently!"
+msgid "Champion - CoderDojo Champions are those who take the lead in organising a Dojo and ensure each session runs smoothly. Champions can be one person or it can be a few people who Co-Champion a Dojo to share the work. Champions should join their Dojo on Zen to set up events, award badges and to contribute to the CoderDojo Forums."
```

Comment:
Absolutely - drop, intended as a title (which I assume is somewhere else :))

On 9 September 2016 at 14:05, Wardormeur <notifications@github.com> wrote:

> In web/locale/en_US/messages.po
> <https://github.com/CoderDojo/cp-zen-platform/pull/1000#discussion_r78163028>
> :
>
> > @@ -2696,17 +2696,17 @@ msgstr "No invites found"
> >  msgid "How do I choose?"
> >  msgstr "How do I choose?"
> >
> > -msgid "Champion - A CoderDojo Champion is an individual who spearheads setting up and maintaining a Dojo. Champions do not necessarily have to have the ability to code but possess the skills required to bring together technical mentors and supporters to run a Dojo frequently!"
> > -msgstr "Champion - A CoderDojo Champion is an individual who spearheads setting up and maintaining a Dojo. Champions do not necessarily have to have the ability to code but possess the skills required to bring together technical mentors and supporters to run a Dojo frequently!"
> > +msgid "Champion - CoderDojo Champions are those who take the lead in organising a Dojo and ensure each session runs smoothly. Champions can be one person or it can be a few people who Co-Champion a Dojo to share the work. Champions should join their Dojo on Zen to set up events, award badges and to contribute to the CoderDojo Forums."
>
> thought abt that, need confirmation from @rosalanghammer
> <https://github.com/rosalanghammer> as i just copy-pasted the content
> from her description :D
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CoderDojo/cp-zen-platform/pull/1000/files/6a4e9ad2106e3e4a1ae716d4b7ce87d469cd5fd5#r78163028>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMO7ILLr4ibmlywCShwBrO7khhj9OD3zks5qoT1wgaJpZM4J47b->
> .
>



-- 
*Rosa Langhammer*
Reporting Lead
@ CoderDojo Foundation
+353 872686 <+353+851088135>779 | @RosaLanghammer | Skype: rosa.langhammer l
<https://mail.google.com/mail/u/0/#UNIQUE_ID_SafeHtmlFilter_> Github:
rosalanghammer

-- 
*CoderDojo are celebrating 5 years since our 1st Dojo*
*See how we've grown since 2011 
<https://coderdojo.com/news/2016/07/23/celebrating-5-years-of-coderdojo/>*

Get tickets to attend our 5th Birthday on September 17th here 
<https://www.eventbrite.ie/e/coderdojo-5th-birthday-celebration-tickets-27217016830?aff=es2>

*Start your own Dojo <https://goo.gl/6xnjlH> and join our global network **of 
more*
*than 1090 Dojos spanning 63 countries!*

-- 
Hello World Foundation operating as the CoderDojo Foundation is a company 
limited by guarantee not having a share capital with charitable status, CHY 
20812. Registered in Dublin, Ireland, No 524255. Registered Office: 
Dogpatch Labs, Unit 1, CHQ Building, Custom House Quay, Dublin 1, Ireland.